.changeset/pretty-seahorses-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-t3-app": patch
```

Comment:
I don't think we need changesets for PRs that don't change the CLI or outputted application